plugins/producer_api_plugin/producer_api_plugin.cpp

```diff
@@ -104,6 +109,18 @@ void producer_api_plugin::plugin_startup() {
             INVOKE_R_V(producer, get_greylist), 201),
        CALL_WITH_400(producer, producer, get_whitelist_blacklist,
             INVOKE_R_V(producer, get_whitelist_blacklist), 201),
+       CALL_WITH_400(producer, producer, set_whitelist_blacklist,
+            INVOKE_V_R(producer, set_whitelist_blacklist, producer_plugin::whitelist_blacklist), 201),
+       CALL_WITH_400(producer, producer, get_integrity_hash,
+            INVOKE_R_V(producer, get_integrity_hash), 201),
+       CALL_ASYNC(producer, producer, create_snapshot, producer_plugin::snapshot_information,
+            INVOKE_R_V_ASYNC(producer, create_snapshot), 201),
```

Comment:
Merge conflicts: `set_whitelist_blacklist`, `get_integrity_hash`, `get_integrity_hash` should be removed from here. They are not safe to read-only transactions and defined from line 147. Sorry for the trouble my PR causes.

Reply:
I checked out your branch and did a test. Confirmed the about problem caused CICD failures:

error 2023-03-16T13:52:42.400 nodeos    main.cpp:161                  main                 ] 3110006  plugin_config_exception: Incorrect plugin configuration
http url /v1/producer/create_snapshot is not unique
    {"u":"/v1/producer/create_snapshot"}
    nodeos  http_plugin.cpp:370 add_handler